src/library/masq.js

```diff
@@ -66,6 +67,13 @@ const openOrCreateDB = (name) => {
   return createPromisifiedHyperDB(name)
 }
 
+const dispatchMasqError = (errorCode) => {
+  const Event = new CustomEvent('MasqError', {
```

Comment:
should be event with lowercase first letter to not confuse it with the Event class